Added an export of orders

enesaktay opened the pull request.

stefandoorn commented:
It would be nicer to split the adjustment to the `DateTimeToStringHandler` into a separate PR. Strictly seen it's unrelated to the order exporter and can be handled/tested/merged separately :)

stefandoorn commented on src/Resources/config/services_export_csv.yml:
```diff
@@ -17,3 +17,13 @@ services:
+            - ["Id", "Number" ,"Notes", "State", "Checkout_completed_at", "Items_total", "Adjustments_total", "Total", "Created_at", "Updated_at", "Currency_code", "Locale_code", "Checkout_state", "Payment_state", "Shipping_state", "Token_value", "Customer_ip"]
+            - "@sylius.exporters_transformer_pool"
+        tags:
+            - { name: sylius.exporter, type: sylius.order, format: csv }
```
Missing newline here :)

enesaktay replied:
I don't see where where the newline is missing, the first one was already without a newline so i copied the style :)

enesaktay replied:
Nvm, @lsmith77 clarified what was said to me, fixed it!

stefandoorn commented on src/Resources/config/services_export_excel.yml:
```diff
@@ -17,3 +17,13 @@ services:
+            - ["Id", "Number" ,"Notes", "State", "Checkout_completed_at", "Items_total", "Adjustments_total", "Total", "Created_at", "Updated_at", "Currency_code", "Locale_code", "Checkout_state", "Payment_state", "Shipping_state", "Token_value", "Customer_ip"]
+            - "@sylius.exporters_transformer_pool"
+        tags:
+            - { name: sylius.exporter, type: sylius.order, format: xlsx }
```
Same here (missing newline) :)

lsmith77 commented:
one question is how elaborately we want to test each type. i think so far we duplicated the bulk of the tests. not sure if this makes sense in the long run.

enesaktay commented:
as the commit message implies I added the userdata and productlist to the order export and removed the WIP in the title as it should now work as wished

enesaktay commented:
has unwanted commits in it, #107 is the new PR without the unwanted commits
